README.md

```diff
@@ -41,6 +41,11 @@ The recommended way to configure the library is to statically initialize it:
 Recurly.Configuration.SettingsManager.Initialize(apiKey, subdomain, pageSize);
 ```
 
+```csharp
+// requestTimeoutMilliseconds is optional and defaults to null
+Recurly.Configuration.SettingsManager.Initialize(apiKey, subdomain, pageSize, requestTimeoutMilliseconds);
```

Comment:
I think I approved too quickly. Isn't this supposed to be
```C#
Recurly.Configuration.SettingsManager.Initialize(apiKey, subdomain, privateKey, pageSize, requestTimeoutMilliseconds);
```

Reply:
If you try it as the example lays it out, you get `error CS1503: Argument 3: cannot convert from 'int' to 'string'` because it thinks that the pageSize should be the string privateKey

Reply:
Good catch, I just pushed the correction. I also fixed it for the pageSize example since that one was wrong as well.